### ISSUE TITLE
 In High contrast white, remove button is not visible when focus is on user control for tag pickers and people pickers

### ISSUE THREAD

**Issue by azra-bandukwala** [microsoft/fluentui#15901]
User Experience:

Visual users are impacted as elements in the page are not visible when focus is on user control in high contrast white.

 

Test Environment:

OS: Windows 10

Version: 2004(OS Build 19577.1000)

Browser: Edge - Dev (84.0.516.1)

URL: https://portal-sdf.office.com/AdminPortal/Home#/featureexplorer

DT Test Environment:

OS: Windows 10

Version: 2004(OS Build 20206.1000)

Browser: Edge Dev 

Browser Version: 87.0.634.0 (Official build) dev (64-bit)

Pre-Requirements :

1) Turn on high contrast White Mode

Repro Steps:

Step 1: Open above URL in Edge Dev browser.

Step 2: Navigate to “Use AI to learn about org view button and activate it

Step 3: Navigate to "Get Started" control and activate it.

Step 4: Navigate to Topic Permissions page by filling all the required fields

Step 5: Navigate to and select "Only selected people" radio button and add user.

Step 6: Observe whether remove button is visible when focus is on user control

 

Actual:

In High contrast white, remove button is not visible when focus is on user control

Expected:

In High contrast white, remove button should be visible when focus is on user control

Note:

1) Issue repro throughout the application

2) Edit button and radio buttons look like static text in HC modes

3) Also Selected Radio buttons are not visible as selected in HC modes

4) Boundary is not available for Cancel/Back buttons in high contrast theme.

MAS Reference

MAS 4.3.1 – No Disruption of Accessibility Features
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B672C8A8B-ED4F-463F-868F-E8131CAF9B74%7D&file=MAS%204.3.1%20%E2%80%93%20No%20Disruption%20of%20Accessibility%20Features.docx&action=default&mobileredirect=true

Parent bug link- https://onedrive.visualstudio.com/CSI/_workitems/edit/879979

![MAS4 3 1_CloseButton_Visible](https://user-images.githubusercontent.com/58192896/98723130-cf727600-2346-11eb-8c89-93fdc0513079.jpg)

**Comment by paulgildea** [comment]
@azra-bandukwala Thanks for filing this issue. Could you specify what version of the package this product is using?

@behowell Please determine if this repos on the website - if so, then we know it's in the latest.

@xugao FYI.

**Comment by azra-bandukwala** [comment]
@paulgildea we are using  "office-ui-fabric-react": "7.135.0"
